src/model/immutable/ContentState.js

```diff
@@ -59,6 +59,32 @@ class ContentState extends ContentStateRecord {
     return this.getBlockChildren('');
   }
 
+  getBlockDescendants(): Object {
```

Comment:
also refactor this algorithm since at its current state is not correct/performant